### DISCUSSION TITLE
Ice 0.11.0 (Beta 3)

### DISCUSSION THREAD

**Discussion by jordanbaird** [jordanbaird/Ice#347] [Announcements]
## What's New
* Brand new UI.

## Bug Fixes
* Fixed a timeout that can occur when moving menu bar items while running certain apps that modify system events.
* Improved compatibility with macOS 15.

## Known Issues
* Applying menu bar item spacing can occasionally fail. This can be resolved by logging out.
* Ice's menu bar item does not respect spacing changes. This can be resolved by restarting Ice.
* Menu bar item search field may not receive initial focus.

**Full Changelog**: https://github.com/jordanbaird/Ice/compare/0.11.0-beta.2...0.11.0-beta.3

<hr /><em>This discussion was created from the release <a href='https://github.com/jordanbaird/Ice/releases/tag/0.11.0-beta.3'>Ice 0.11.0 (Beta 3)</a>.</em>

**Comment by bobagit** [comment]
Apologies in advance if this is in the wrong place — I have macOS Sequoia 15.1 and the app Ice doesn't load (so I cannot get to settings). The app functions, but I cannot get to the preferences menu as the app doesn't load. I have the Ice Beta 3 installed, and I tried the public release as well, both do not show the preferences menu.

**Reply by jordanbaird** [reply]
What are the steps you're following to try to get into the settings?

**Reply by jordanbaird** [reply]
You have to right click on Ice's menu bar item to get to settings.

**Reply by snapesworstmemory** [reply]
I have the same issue. Ice is running, but I've hidden the Ice menu bar icon, so I'm unable to access the settings to change them.
How do I access the Ice settings if the Ice menu bar item is hidden?

**Reply by jordanbaird** [reply]
@snapesworstmemory Either open the app again through spotlight or by double clicking its app icon, or right click an empty area in the menu bar (actual right click, not control click).

**Reply by snapesworstmemory** [reply]
Thanks!

**Comment by daschles** [comment]
right click on icon doesn’t give me settings, just shows the extended list
of apps. Now I seem to have no way to get into settings at all. MacBook Air
M3 running 15.0 ice 0.11.1
thanks
——
David Schlesinger

**Reply by jordanbaird** [reply]
If you're trying to right click using the control key, Ice currently doesn't support this, but there's an issue open to track adding it as a feature. In the meantime, you can right click by clicking with two fingers instead of one.

**Comment by bobagit** [comment]
A quick note, on my computer, even with the trackpad secondary click option disabled, option click always acts as a right click, this works throughout the system, but when you option click the ice logo, it does not display a dialogue below. in short, Right clicking the logo using option click on a laptop does not work. Sent from my iPhoneOn Sep 16, 2024, at 8:50 AM, daschles ***@***.***> wrote:﻿
 right click on icon doesn’t give me settings, just shows the extended list

of apps. Now I seem to have no way to get into settings at all. MacBook Air

M3 running 15.0 ice 0.11.1

thanks

——

David Schlesinger

**Reply by jordanbaird** [reply]
Fairly sure you mean control click, right? That's always been the macOS alternative to right click, never option. There is an issue (#343) open for that.

**Comment by bobagit** [comment]
You are correct. I meant Control click. I was able to access settings with two finger tap (labeled as Secondary click in Sequoia settings). Sent from my iPhoneOn Sep 16, 2024, at 11:22 AM, Jordan Baird ***@***.***> wrote:﻿
Fairly sure you mean control click, right? That's always been the macOS alternative to right click, never option. There is an issue (#343) open for that.

**Comment by jordanbaird** [comment]
We're no longer on this beta release, but people are still keeping this thread active. If you have further issues, please report them in the appropriate place.
